source/index.html

```diff
@@ -3,7 +3,103 @@
+			<li class="advantages__item advantages__item--stile">Скандинавский стиль по российской цене</li>
+			<li class="advantages__item advantages__item--like">Увеличивает лайки на фотографиях</li>
+			<li class="advantages__item advantages__item--hand">Связано вручную с любовью и умилением</li>
+			<li class="advantages__item advantages__item--support">Поддержка отечественного производителя</li>
+			<li class="advantages__item advantages__item--box">Поставляется в подарочной упаковке</li>
+		</ul>
+		</div>
+	</section>
+	<section class="reviews">
+		<div class="reviews__wrapper page__centerer">
+				<h2 class="reviews__title">Отзывы</h2>
+			<div class="reviews__list">
+				<blockquote class="reviews__item reviews__item--current">
+					<p class="reviews__text">Я молодая мама, но всё равно фотографии не набирали и близко такого количества лайков, как 	у популярных инстамамочек. В отчаянии, я накупила аксессуаров и игрушек в Мишке, и мои фотографии сразу стали более стильными, а также набирают больше лайков!</p>
+					<span class="reviews__author-name">Анастасия Красильникова</span>
+					<span class="reviews__author-email">@misssssiskras</span>
+				</blockquote>
+			</div>
+			<div class="reviews__list">
+				<blockquote class="reviews__item">
+					<p class="reviews__text">Я сестра молодой мамы, но всё равно фотографии не набирали и близко такого количества лайков, как у популярных инстамамочек. В отчаянии, я накупила аксессуаров
+						и игрушек в Мишке, и мои фотографии сразу стали более стильными, а также набирают
+						больше лайков!</p>
+					<span class="reviews__author-name">Екатерина Красильникова</span>
+					<span class="reviews__author-email">@misssssiskras2</span>
+				</blockquote>
+			</div>
+			<div class="reviews__list">
+				<blockquote class="reviews__item reviews__item--active">
+					<p class="reviews__text">Я двоюродная сестра молодой мамы, но всё равно фотографии не набирали и близко такого количества лайков, как у популярных инстамамочек. В отчаянии, я накупила аксессуаров 	и игрушек в Мишке, и мои фотографии сразу стали более стильными, а также набирают больше лайков!</p>
+					<span class="reviews__author-name">Елена Красильникова</span>
+					<span class="reviews__author-email">@misssssiskras3</span>
+				</blockquote>
+			</div>
+			<button class="reviews__toogle reviews__toogle--before"></button>
+			<button class="reviews__toogle reviews__toogle--after"></button>
+		</div>
+	</section>
+	<section class="contacts">
+		<div class="contacts__wrapper page__centerer">
+			<h2 class="contacts__title">Контакты</h2>
+			<p class="contacts__email">
+				<span class="contacts__email-label">e-mail:</span>
+				<span class="contacts__email-email">info@mimimishkashop.ru</span>
+			</p>
+			<address class="contacts__address">
+				<span class="contacts__address-label">адрес:</span>
+				<span class="contacts__address-address">г. Санкт-Петербург ул. Большая Конюшенная д. 19/8, офис 101</span>
+			</address>
```

Comment:
Лучше в тег `address` помещать только сам адрес, без упоминания того, что это адрес.
```suggestion
			<p class="contacts__address">
				<span class="contacts__address-label">адрес:</span>
				<address class="contacts__address-address">г. Санкт-Петербург ул. Большая Конюшенная д. 19/8, офис 101</address>
			</p>
```
Но помним, что лучше `p` не использовать в сетках, так что и тут и у email лучше `div` использовать вместо параграфа. Или вообще все контактные данные можно сделать через `dl>dt+dd`.